fixieai/cli/agent/commands.py

```diff
@@ -14,6 +16,14 @@
 from fixieai.cli.agent import agent_config
 from fixieai.cli.agent import loader
 
+# Regex pattern to match valid entry points: "module:object"
+VAR_NAME_RE = r"(?![0-9])\w+"
+ENTRY_POINT_PATTERN = re.compile(rf"^{VAR_NAME_RE}(\.{VAR_NAME_RE})*:{VAR_NAME_RE}$")
+# The agent template main.py file
+AGENT_TEMPLATE_URL = (
+    "https://raw.githubusercontent.com/fixie-ai/fixie-examples/main/agents/template.py"
```

Comment:
as discussed let's remove everything but the basics from the template

Reply:
Will do in the fixie example repo